org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug381381TestLanguageGrammarAccess.java

```diff
@@ -66,18 +66,18 @@
 		private final Keyword cREPLACEKeyword_1_2_3 = (Keyword)cGroup_1_2.eContents().get(3);
 		
 		//CopyFieldNameToVariableStmt:
-		//	'FIELD-NAME-TO-VARIABLE' (',' 'SCREEN' '=' '(' line=INT ',' column=INT ')' & ',' 'VAR' '=' name=ID & (',' 'TYPE' '='
-		//	'REPLACE')?);
+		//	'FIELD-NAME-TO-VARIABLE' ( ',' 'SCREEN' '=' '(' line=INT ',' column=INT ')' & ',' 'VAR' '=' name=ID & (',' 'TYPE' '='
+		//	'REPLACE')? );
 		@Override public ParserRule getRule() { return rule; }
 		
-		//'FIELD-NAME-TO-VARIABLE' (',' 'SCREEN' '=' '(' line=INT ',' column=INT ')' & ',' 'VAR' '=' name=ID & (',' 'TYPE' '='
-		//'REPLACE')?)
+		//'FIELD-NAME-TO-VARIABLE' ( ',' 'SCREEN' '=' '(' line=INT ',' column=INT ')' & ',' 'VAR' '=' name=ID & (',' 'TYPE' '='
+		//'REPLACE')? )
 		public Group getGroup() { return cGroup; }
 		
 		//'FIELD-NAME-TO-VARIABLE'
 		public Keyword getFIELDNAMETOVARIABLEKeyword_0() { return cFIELDNAMETOVARIABLEKeyword_0; }
 		
-		//(',' 'SCREEN' '=' '(' line=INT ',' column=INT ')' & ',' 'VAR' '=' name=ID & (',' 'TYPE' '=' 'REPLACE')?)
+		//( ',' 'SCREEN' '=' '(' line=INT ',' column=INT ')' & ',' 'VAR' '=' name=ID & (',' 'TYPE' '=' 'REPLACE')? )
```

Comment:
@ArneDeutsch Also the space after the opening parentheses.

Reply:
Should be fixed with https://github.com/eclipse/xtext-core/issues/1489

Reply:
problem still seems to be there

Reply:
see above